geopy/geocoders/base.py

```diff
@@ -45,6 +46,11 @@
 DEFAULT_TIMEOUT = 1
 DEFAULT_WKID = 4326
 
+
+def get_default_user_agent():
+    return "geopy/" + gu.get_version()
```

Comment:
In my experience it is much nicer to test if dynamic things are not fixed at import time (constant), but at call time (function). It is then very easy to mock this function within the test to a known test return value, e.g: "mytestagent/1.2.3". At least I haven't figured out how to do that with a constant...
What would be the benefit of a constant?